daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DaffodilParseXMLReader.scala

```diff
@@ -194,8 +198,10 @@ class DaffodilParseXMLReader(dp: DataProcessor) extends DFDL.DaffodilParseXMLRea
    *
    * @return SAXInfosetOutputter object with or without blob Attributes set
    */
-  private def createSAXInfosetOutputter(xmlReader: DaffodilParseXMLReader): SAXInfosetOutputter = {
-    val sioo = new SAXInfosetOutputter(xmlReader)
+  private def createSAXInfosetOutputter(xmlReader: DaffodilParseXMLReader,
+    namespacesFeature: Boolean,
+    namespacePrefixesFeature: Boolean): SAXInfosetOutputter = {
```

Comment:
Minor inconsisency here. We pass in these two features to this funciton (which are private members) but don't pass in the blob properties (which are also private members). So it's not clear why some things are passed in and other very simialr things aren't.

I guess really, we don't even need to pass in anything to this function since everything it accesses is a private member of the class.

In fact, this function is only called in one place, and it's pretty small. Thoughts on just moving this function  logic to the on place it's called and not having to worry about passing anything in? So the one ``parse`` function is responsible for creating the infoset outputter and gett the properties/features values?

Reply:
I think we'd chosen to do it this way, because we wanted to guarantee that once the function was called, no changes could be made to those particular features so they'd be read-only on parse

Reply:
I'm talking about passing things into the createSAXInfosetOutputter function. We pass in the features as parameters, but not the properties. We could easily pass in the properties, or we could pass in nothing and just have this function access the private memebers directly. Not sure there's reason to pass in features but not properties. And if list of features started to grow big, then passing them in would be a pain. 

I also suggest this fucntion is small enough and is only called in one place that it might just be worth moving the logic in the the parse function. Then you don't even need to worry about what gets passe in vs directly sued as member vars.

Reply:
Ah I see. Makes sense, moved functionality to the parse and deleted the function